docs/network.txt

```diff
@@ -3,37 +3,41 @@ IP Space
 
 Public IP space is 140.211.10.128/25, in a single VLAN shared with RTEMS.
 
-Buildbot is allocated 140.211.10.224/27 out of that /25.
+Buildbot is allocated 140.211.10.224/27 out of that /25:
+    140.211.10.225 -- 140.211.10.254
+
+(Misha: it looks like we are given the whole /25, so 140.211.10.224 can also be used.)
```

Comment:
Yes, in the note it should be `/27`.

The reason behind this note is that I'm not sure if it was a "human" split.  So I'd like to get a confirmation for that :)  (The network configuration of service hosts suggests it is a human split.)